Sources/ShopifyCheckoutSheetKit/CheckoutViewController.swift

```diff
@@ -66,9 +66,9 @@ public struct CheckoutSheet: UIViewControllerRepresentable, CheckoutConfigurable
     var delegate = CheckoutDelegateWrapper()
     var options: CheckoutOptions?
```

Comment:
I'm not opposed to moving the checkoutoptions to a modifier, but wonder if it'd be easier/make sense for the nilability to live on the properties rather than the options itself? 

`CheckoutOptions.authentication` is already nilable, so we could make the options required here, and just set the property on the auth modifier
I think that may simplify things


```suggestion
    var options: CheckoutOptions = .init()
```